lib/IDE/Refactoring.cpp

```diff
@@ -1909,8 +1909,9 @@ class ExpandableAssignTernaryExprInfo: public ExpandableTernaryExprInfo {
   IfExpr *getIf() {
     if (!Assign)
       return nullptr;
-
-    return dyn_cast<IfExpr>(Assign->getSrc());
+    if (auto *Src = Assign->getSrc())
```

Comment:
I think you could just do `return dyn_cast_or_null<IfExpr>(Assign->getSrc);`

Reply:
Done.